c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -918,6 +918,22 @@ mfxStatus MfxC2DecoderComponent::InitSession()
         return MFX_ERR_UNKNOWN;
     }
 
+    cfg[2] = MFXCreateConfig(m_mfxLoader);
+    if (!cfg[2]) {
+        ALOGE("Failed to create a MFX configuration");
+        MFXUnload(m_mfxLoader);
+        return MFX_ERR_UNKNOWN;
+    }
+
+    cfgVal[2].Type = MFX_VARIANT_TYPE_U32;
+    cfgVal[2].Data.U32 = DRM_RENDER_NODE_NUM;
```

Comment:
I see DRM_RENDER_NODE_NUM is set to 128. Does it work for iGPU+dGPU?  

Thanks,
Tianmi

Reply:
Yes, in dual GPU case, only use iGPU